addon/components/bs-button.hbs

```diff
@@ -1,9 +1,10 @@
 <button
   disabled={{this.__disabled}}
   type={{this.buttonType}}
-  class="btn {{if this.active "active"}} {{if this.block "btn-block"}} {{this.sizeClass}} {{this.typeClass}}"
+  class="btn {{if @active "active"}} {{if this.block "btn-block"}} {{bs-size-class "btn" @size}} {{bs-type-class "btn" @type default=(if (has-bootstrap-version 3) "default" "secondary") outline=@outline}}"
```

Comment:
As `this.block` is just an alias with default property of `false`, `{{if this.block "btn-block"}}` should be the same as `{{if @block "btn-block}}`.

```suggestion
  class="btn {{if @active "active"}} {{if @block "btn-block"}} {{bs-size-class "btn" @size}} {{bs-type-class "btn" @type default=(if (has-bootstrap-version 3) "default" "secondary") outline=@outline}}"
```

Reply:
See above.